test/support/fixtures/arguments_schema.ex

```diff
@@ -7,6 +7,11 @@ defmodule Absinthe.Fixtures.ArgumentsSchema do
     false: "NO"
   }
 
+  scalar :any do
+    parse fn value -> {:ok, value} end
```

Comment:
Wonder if I could even leave these out? if I dont provide these maybe we just coerce them into built-in types?